### ISSUE TITLE
Mysterious hang

### ISSUE THREAD

**Issue by znicholls** [jupyterhub/the-littlest-jupyterhub#338]
As raised in #325, I somehow managed to break my server, but a day later it worked after re-building the machine. I can't reproduce the issue and am not sure what I did, I might have installed and then uninstalled Apache so was blocking the port, I might have had the wrong security settings when I booted the VM, I might have somehow bungled my ports as I was playing around with firewalls, not sure... 

Would it be worth adding a 'Have you tried re-booting from scratch?' reminder to the troubleshooting page of custom servers?

**Comment by yuvipanda** [comment]
Thanks for opening this, @znicholls! When you say 're-booting', did you turn it off and on or delete and re-create? Knowing that would be useful.

Related discussion at #128

**Comment by znicholls** [comment]
Delete and re-create

**Comment by yuvipanda** [comment]
@znicholls makes sense! I agree we should document that. Would love it if you can work on a PR to insert it. Is that something you're up for doing?

**Comment by znicholls** [comment]
yep will do in #325

**Comment by aboutaaron** [comment]
I think it also be worth including _how_ to do this on custom server, e.g., including steps/links to docs for taking down jupyterlab or which commands to run to restart the server.
